.github/workflows/push.yml

```
@@ -81,12 +87,12 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-java@v4
         with:
-          distribution: 'zulu' # See 'Supported distributions' for available options
-          java-version: '17.0'          
+          distribution: "zulu" # See 'Supported distributions' for available options
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)